front/src/modules/trainschedule/components/ManageTrainSchedule/PowerRestrictionsSelector.tsx

```diff
@@ -203,6 +140,39 @@ const PowerRestrictionsSelector = ({
         {!isEmpty(rollingStockPowerRestrictions) ? (
           <>
             <p className="mb-1 mt-1">{t('powerRestrictionExplanationText')}</p>
+            {totalPowerRestrictionWarnings > 0 && (
+              <div className="border border-warning rounded p-3 my-3">
+                <div className="d-flex align-items-center mb-2">
+                  <Alert />
+                  <span className="ml-2">
+                    {totalPowerRestrictionWarnings > 1
+                      ? t('warningMessages.inconsistent_many')
+                      : t('warningMessages.inconsistent_one')}
+                  </span>
+                </div>
+                {Object.keys(powerRestrictionsWarnings).map((powerRestrictionCategory) => (
```

Comment:
You can use Object.entries to directly get the key and value:
```suggestion
                {Object.entries(powerRestrictionsWarnings).map(
                  ([warningCategory, warningsByCategory]) => 
```

Reply:
Good point, done.